Author: Steven Chamberlain <steven@pyro.eu.org>
Description: The missing sys/stat.h prototype leads to an implicit declaration
 of fchmod(), which uses signed int while kFreeBSD expects unsigned int, which
 results in a wrong bitmap, which leads to a build failure.
Bug-Debian: http://bugs.debian.org/654767

--- a/fig2dev/dev/genpstex.c
+++ b/fig2dev/dev/genpstex.c
@@ -37,6 +37,7 @@
  * Jose Alberto.
  */
 
+#include <sys/stat.h>
 #include "fig2dev.h"
 #include "genps.h"
 #include "genpdf.h"
